plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/reports/class-wc-tests-reports-orders-stats.php

```diff
@@ -11,6 +11,8 @@
 
 /**
  * Class WC_Admin_Tests_Reports_Orders_Stats
+ * Excluding it from code coverage until we refactor this test.
+ * @group run-in-seprate-process
```

Comment:
Perhaps typo here, `run-in-separate-process`?

Reply:
Thank you for catching it! Yes, it's a typo. Updated in https://github.com/woocommerce/woocommerce/pull/32409/commits/8632950ab8a3d6a47d37801ca5aa8ee10330ef4f